nimble/transport/socket/src/ble_hci_socket.c

```diff
@@ -420,15 +422,15 @@ ble_hci_sock_rx_msg(void)
 }
 
 static void
-ble_hci_sock_rx_ev(struct os_event *ev)
+ble_hci_sock_rx_ev(struct ble_npl_event *ev)
 {
     int rc;
 
     rc = ble_hci_sock_rx_msg();
     if (rc == 0) {
-        os_eventq_put(&ble_hci_sock_state.evq, &ble_hci_sock_state.ev);
+        ble_npl_eventq_put(&ble_hci_sock_state.evq, &ble_hci_sock_state.ev);
     } else {
-        os_callout_reset(&ble_hci_sock_state.timer, OS_TICKS_PER_SEC / 100);
+        ble_npl_callout_reset(&ble_hci_sock_state.timer, OS_TICKS_PER_SEC / 100);
```

Comment:
explicit usage of `OS_TICKS_PER_SEC` should be removed in favor of NPL APIs

Reply:
Agreed.  This got missed -- the older osal used these defines directly and I missed the change to new NPL_ names.